lib/core/src/display/symbol/buffer.rs

```diff
@@ -306,7 +306,7 @@ impl<T> Builder<T> {
 // ========================
 
 // TODO The following code should be refactored to use the new macro `eval-tt`
-// TODO engine. Some utils, like `cartesian` macro should also be refactored
+// TODO rendering. Some utils, like `cartesian` macro should also be refactored
```

Comment:
It doesnt make sense now 😂 What is `eval-tt` rendering? :O Btw, Danilo, I suggest using IntelliJ refactoring, it works really nice and saves ton of time.

Reply:
Fun fact: I used IntelliJ refactoring everywhere lol I don't know what happened here.

I renamed it back to `engine`.

Reply:
Oh, it was done by intellij? Thats funny :D